translations/party+intl-icu.da.xlf

```diff
@@ -81,6 +81,14 @@
         <source>Landlord (administrator)</source>
         <target>Udlejer (administrator)</target>
       </trans-unit>
+      <trans-unit id="xwmzKet" resname="Identifier type">
+        <source>Identifier type</source>
+        <target state="translated">Id type</target>
```

Comment:
Sammensatte navneord skrives på dansk altid i ét ord (og med bindestreg når der sammensættes med en forkortelse), dvs. det skal være “Id-type”.

Reply:
Fixed in 00acbb7fa8f6e10cb0b843f6d3ef8483c8a14dc3.